src/views/Home/Home.jsx

```diff
@@ -1,33 +1,55 @@
-import React, { useEffect } from 'react';
-import { useLocation } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { useTracking } from 'react-tracking';
 
 import NoResults from '../../components/molecules/no-results/no-results';
-import { getKeyValueFromQueryString } from '../../utils';
+import SearchResultsList from '../../components/molecules/search-results-list/search-results-list';
+
+import { getSearchResults } from '../../services/api/actions/';
+import { useURLQuery, useCustomQuery } from '../../hooks';
 import { useStateValue } from '../../store/store';
-import { useTracking } from 'react-tracking';
 
 const Home = () => {
-	const location = useLocation();
-	const { search } = location;
-	const [{ canonicalHost, title, siteName }] = useStateValue();
-	const keyword = getKeyValueFromQueryString('swKeyword', search);
+	const urlQuery = useURLQuery();
+	const [{ canonicalHost, siteName, title, language }] = useStateValue();
+	const [stateSearchResults, setStateSearchResults] = useState();
+	const [pageunit, setPageunit] = useState(20);
+	const keyword = urlQuery.get('swKeyword');
 	const tracking = useTracking();
+	const searchResults = useCustomQuery(
+		getSearchResults({ language, keyword, pageunit }),
+		!!keyword
+	);
+	// Set hasResults should there be results returned for any search
+	const hasResults = !!keyword && stateSearchResults?.result?.length > 0;
+
+	useEffect(() => {
+		if (!searchResults.loading && searchResults.payload) {
+			tracking.trackEvent({
+				event: 'SiteWideSearchApp:Load:Results',
+				metaTitle: `${title} - ${siteName}`,
+				name: `${canonicalHost.replace('https://', '')}${
+					window.location.pathname
+        }${window.location.search}`,
+				numberResults: searchResults.payload.totalResults,
+				pageNum: 1,
```

Comment:
This value should be computed based on the actual page number displayed - based on team discussion, it will be fixed in Seyi's PR

Reply:
@kate-mashkina On second thoughts, it's okay for this to be here for now. It will be handled in the (#2) Results Pager PR as this technically doesn't apply paging yet. So this is fine for this PR.